cypress/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "strict": true,
     "target": "ES5",
     "tsBuildInfoFile": "../node_modules/.cache/tsbuildinfo/cypress", // avoid clobbering the build tsbuildinfo
-    "types": ["cypress", "node"]
+    "types": ["cypress", "node"],
+    "jsx": "react"
```

Comment:
is this necessary?

Reply:
what does this do? why wasn't it happening before?

Reply:
@zzmp @vm no tests were actually using the featureFlags option before (i'm adding one in a follow up diff)

afaict this is needed because we moved the `enum FeatureFlag` into `index.tsx` (from `featureFlags.ts`) and this is just a flag required to support .tsx files